python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -222,17 +222,36 @@ def forward_cuda(
         query: torch.Tensor,
         key: torch.Tensor,
         offsets: Optional[torch.Tensor] = None,
+        layer: Any = None,  # RadixAttention
+        forward_batch=None,
+        save_kv_cache: bool = False,
+        value: Optional[torch.Tensor] = None,
     ) -> Tuple[torch.Tensor, torch.Tensor]:
         if _is_cuda and (self.head_size in [64, 128, 256, 512]):
+            from sglang.srt.model_executor.cuda_graph_runner import get_is_capture_mode
+
+            # import importlib.util
+            # spec = importlib.util.spec_from_file_location("apply_rope_with_cos_sin_cache_inplace", "/sgl-workspace/sglang/sgl-kernel/python/sgl_kernel/elementwise.py")
+            # elementwise = importlib.util.module_from_spec(spec)
+            # spec.loader.exec_module(elementwise)
+            # elementwise.apply_rope_with_cos_sin_cache_inplace(
             apply_rope_with_cos_sin_cache_inplace(
                 positions=positions,
                 query=query,
                 key=key,
                 head_size=self.head_size,
                 cos_sin_cache=self.cos_sin_cache,
                 is_neox=self.is_neox_style,
+                layer=layer,
+                forward_batch=forward_batch,
```

Comment:
maybe we should not pass such objects to this API

what about e.g.

```
def apply_rope_with_cos_sin_cache_inplace(
  ...,
  # in non-fused version we do `k_buffer[loc] = data` etc
  k_buffer: Tensor, v_buffer: Tensor, loc: Tensor,
)
```

and if none, it means we do not save kv cache; if non-none then we need to save